bus-mapping/src/evm/opcodes.rs

```diff
@@ -40,11 +54,149 @@ type FnGenAssociatedOps = fn(
 impl OpcodeId {
     fn fn_gen_associated_ops(&self) -> FnGenAssociatedOps {
         match *self {
+            OpcodeId::STOP => Stop::gen_associated_ops,
+            OpcodeId::ADD => BinaryOpcode::gen_associated_ops,
+            OpcodeId::SUB => BinaryOpcode::gen_associated_ops,
+            OpcodeId::MUL => BinaryOpcode::gen_associated_ops,
```

Comment:
Following the order of opcode's value seems to be more easier to check if all opcodes are included.

```suggestion
            OpcodeId::MUL => BinaryOpcode::gen_associated_ops,
            OpcodeId::SUB => BinaryOpcode::gen_associated_ops,
```